blog/admin.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from blog.models import User, Post, Commentary
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    pass
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    search_fields = ["title"]
+    list_filter = ["owner", "created_time"]
+    list_display = ["title", "owner", "created_time"]
+    ordering = ["-created_time", "owner__username"]
+
+
+@admin.register(Commentary)
+class CommentaryAdmin(admin.ModelAdmin):
+    search_fields = ["user"]
```

Comment:
The 'search_fields' attribute in the 'CommentaryAdmin' class is currently set to ['user']. However, 'user' is a foreign key, so you should specify a field on the related model to search on. For example, if you want to search by the username of the user who created the comment, you can use 'user__username' instead of 'user'.